services/src/main/java/org/exoplatform/officeonline/WOPIService.java

```diff
@@ -60,106 +62,110 @@
 import org.exoplatform.services.wcm.core.NodetypeConstant;
 import org.exoplatform.services.wcm.utils.WCMCoreUtils;
 
+// TODO: Auto-generated Javadoc
```

Comment:
Remove it

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/10/commits/36cfb2d370ee05b1aab1b09e471285054e5971e8